src/v2/layouts/MemberDataGrid/MemberInfoButtons/DesktopButtons.jsx

```diff
@@ -0,0 +1,102 @@
+import styled, { css } from 'styled-components';
+import { faSync } from '@fortawesome/free-solid-svg-icons/faSync';
+import { faCog } from '@fortawesome/free-solid-svg-icons/faCog';
+import { faFile } from '@fortawesome/free-solid-svg-icons/faFile';
+import { Icon } from '../../../components/design/Icon/Icon';
+import { PageRoutes } from '../../../components/providers/RouteProvider/RouteProvider';
+import { Link } from 'react-router-dom';
+import { useContext } from 'react';
+import { StringsContext } from '../../../components/providers/StringsProvider/StringsContext';
+import { PrimaryButton } from '../../../components/design/Button';
+import { faUserPlus } from '@fortawesome/free-solid-svg-icons/faUserPlus';
+import PropTypes from 'prop-types';
+
+export const DesktopButtons = ({ setIsVisibleConfigModal, refreshStudentsDataGrid }) => {
+	const { strings } = useContext(StringsContext);
+	const { NEW_REGISTRATION, FILES } = strings;
+	const { MEMBER_REGISTRATION } = PageRoutes;
+
+	return (
+		<DesktopButtonsBoxStyled>
+			<LinkStyled to={MEMBER_REGISTRATION} margin-right="40px">
+				<PrimaryButton text={NEW_REGISTRATION} margin-right="70px">
+					<Icon icon={faUserPlus} />
+				</PrimaryButton>
+			</LinkStyled>
+			<LinkStyled to={''}>
+				<PrimaryButton text={FILES}>
+					<Icon icon={faFile} />
+				</PrimaryButton>
+			</LinkStyled>
+			<LinkStyled
+				onClick={() => {
```

Comment:
I think we should use it as a callback instead
```suggestion
				onClick={onClickConfigBtn}
```